src/main/java/com/artipie/asto/cache/Remote.java

```diff
@@ -26,14 +26,15 @@
 import com.artipie.asto.Content;
 import com.jcabi.log.Logger;
 import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 import java.util.function.Supplier;
 
 /**
  * Async {@link java.util.function.Supplier} of {@link java.util.concurrent.CompletionStage}
  * with {@link Optional} of {@link Content}. It's a {@link FunctionalInterface}.
  *
- * @since 0.4
+ * @since 0.40
```

Comment:
@olenagerasimova latest version is `0.31`, so I guess proper `@since` tag version is `0.32`

Reply:
@olegmoz hm, in my experience `@sinse` tag usually contains next major version, in our case it's `0.40`

Reply:
@olegmoz common version format is `<major>.<minor>.<patch>`. Next major version for us is `1.0` and next minor is `0.32`. We are releasing minor versions every time, but they happen to have breaking changes. I see how `@since` tag in this case might be `1.0` or `0.32`, but `0.40` just does not make sense to me, sorry

Reply:
@olegmoz ok, agreed, corrected since tags